docs/content/docs/custom-resource/reference.md

```diff
@@ -160,26 +160,41 @@ This page serves as a full reference for FlinkDeployment custom resource definit
 
 ## Status
 
+### CommonStatus
+**Class**: org.apache.flink.kubernetes.operator.crd.status.CommonStatus
+
+**Description**: Last observed common status of the Flink deployment/Flink SessionJob.
+
+| Parameter | Type | Docs |
+| ----------| ---- | ---- |
+| jobStatus | org.apache.flink.kubernetes.operator.crd.status.JobStatus | Last observed status of the Flink job on Application/Session cluster. |
+| error | java.lang.String | Error information about the FlinkDeployment/FlinkSessionJob. |
```

Comment:
The inheritence structure breaks the (not very sophisticated) doc generation logic. We should either fix it here or open a blocker jira to track this

Reply:
I also confused here, what‘s the expect content here? Do we need to display the `FlinkDeploymentReconciliationStatus` fileds include (in the parent) directly and skip the abstract classes

Reply:
I would open another PR after this to fix this problem soon .

Reply:
I think from the users perspective it would be best to skip any abstract classes and only show the “child/leaf” classes in the hierarchy with all fields from the superclasses .

Reply:
Get, I will try to make up the content in this way